theme_monglia/static/src/snippets/s_numbers/000.scss

```diff
@@ -1,11 +1,3 @@
-.s_numbers {
-    .s_number {
-        font-size: 75px;
-        @include media-breakpoint-down(md) {
-            font-size: 60px;
-        }
-    }
-    h6 {
+.s_numbers h6 {
         text-transform: uppercase;
```

Comment:
@qsm-odoo During my check I saw that this uppercase rule doesn't work. Maybe we could juste remove it or we can create a task to fix it
![image](https://github.com/odoo/design-themes/assets/78849981/ae945b89-648a-4958-837b-e9d5136dee81)

Reply:
As you wish, as long as it is done in a dedicated commit (maybe even PR) :+1:

Reply:
Will be fixed in [this task](https://www.odoo.com/web#id=3469717&cids=1&menu_id=4720&action=333&active_id=974&model=project.task&view_type=form) :heavy_check_mark: